azurelinuxagent/ga/cgroupconfigurator.py

```diff
@@ -478,6 +469,9 @@ def agent_enabled(self):
         def extensions_enabled(self):
             return self._extensions_cgroups_enabled
 
+        def cgroup_v2_enabled(self):
```

Comment:
rename to using_cgroup_v2 or similar?

currently cgroup_v2_enabled() can return True when enabled() returns False. That is just weird.